app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddPathSurface.kt

```diff
@@ -9,7 +9,7 @@ import de.westnordost.streetcomplete.data.osm.mapdata.OverpassMapDataAndGeometry
 import de.westnordost.streetcomplete.settings.ResurveyIntervalsStore
 
 class AddPathSurface(o: OverpassMapDataAndGeometryApi, r: ResurveyIntervalsStore)
-    : SimpleOverpassQuestType<String>(o) {
+    : SimpleOverpassQuestType<DetailSurfaceAnswer>(o) {
 
     override val tagFilters = """
```

Comment:
I think something is missing here in the filter: Looking for surface~paved|unpaved. Or is this deliberate?